src/api/spec/models/binary_release_spec.rb

```diff
@@ -6,10 +6,26 @@
       'disturl' => '/foo/bar',
       'supportstatus' => 'foo',
       'binaryid' => '31337',
-      'buildtime' => '1640772016'
+      'buildtime' => '1640772016',
+      'binaryarch' => 'i386',
+      'name' => 'foo'
     }
   end
 
+  describe '.update_binary_releases' do
```

Comment:
@vpereira just thinking if it might make sense to create a `BinaryRelease` record upfront, to see if it really gets updated. If I understand correctly thats what this method is meant for, to check the backend for existing binaries and then to update both on the frontend and backend.

Reply:
So what I mean is that now we see that it successfully created a new record since there is no matching one, but we don't cover that it correctly checks existing records and replaces them with an updated version if exists